docs/automated-testing/e2e-testing/recipes/postman-testing.md

```diff
@@ -0,0 +1,230 @@
+# Postman and Newman
```

Comment:
can you add an overview that describes the flow so that the reader understands what this is about and can decide whether to continue reading or not

Reply:
Pushed.